.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Bug report
+about: Something is broken and you want help fixing it
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+If you want your issue to be replied to, writing a detailed thought-out issue makes it more appealing to respond to. Otherwise, your issue will only serve as a forum for other users experiencing the issue to comment on. Replies are not to be expected by default.
```

Comment:
What do you think of having a Flake template here where we can ask users to try to reproduce the issues using it? This should make reproduction of issues much easier.

Reply:
Sounds good, but I'm not sure how you want to do that so feel free to post here/push to the branch.

Reply:
Sure, I can work this later on once I have free time.

Reply:
```suggestion
If you want your issue to be replied to, writing a detailed thought-out issue makes it more appealing to respond to. Otherwise, your issue will only serve as a forum for other users experiencing the issue to comment on. Replies are not to be expected by default.

To help debugging your issue, you can try to replicate it using a minimal example with the following `flake.nix` file:

{
  description = "Test";

  inputs = {
    nix-doom-emacs.url = "github:nix-community/nix-doom-emacs";
    flake-utils.url = "github:numtide/flake-utils";
  };

  outputs = { self, nix-doom-emacs, flake-utils, ... }@inputs:
    flake-utils.lib.eachDefaultSystem (
      system: {
        packages.default = nix-doom-emacs.package.${system} {
          doomPrivateDir = ./doom.d;
        };
      }
    );
}
```